### ISSUE TITLE
Different Results for Labram linear probe compared to the paper on TUAB

### ISSUE THREAD

**Issue by saarangp** [xw1216/EEG-FM-Bench#5]
Hello! 

I'm finding a fairly large difference in the B-Acc for TUAB for Labram. 

My config is simply this, and I get a B-Acc of ~75% on the eval set, compared to 60.40%. Is there a mismatch with the configurations? 

```
model_type: 'labram'                                                          
                                                                                
model:                                                                        
  pretrained_path: './labram-base.pth'
                                                                              
data:                                                                         
  batch_size: 32                                                              
  datasets:                                                                   
    tuab: 'finetune'                                                          
                                                                              
training:                                                                     
  max_epochs: 30                                                              
  freeze_encoder: true  # Linear probe                                        
  max_lr: 4e-4                                                                
                                                                              
logging:                                                                      
  output_dir: './assets/run/labram_tuab'                                      
  experiment_name: 'labram_tuab'   
  use_cloud: False
  
```

**Comment by xw1216** [comment]
Hi! Thank you for checking this. In my implementation, freeze_encoder freezes the encoder weights, but the training loop still calls model.train(), so any dropout layers in the encoder remain active during head fine-tuning. The paper follows a consistent protocol across paradigms with non-zero dropout. Since this protocol choice can affect downstream performance, I’ll make the frozen-encoder train/eval behavior explicit in the repo docs for reproducibility.
